### PR TITLE
Add demo day 8 post

### PR DESCRIPTION
**Pull request by jdebacker**
This PR adds a blog post for Demo Day 8: Jason DeBacker on contributing to PSL projects.

Will remove [WIP] when youtube video live and can get correct link.

### CODE REVIEW

**Comment by jdebacker** [comment]
Screenshot:

Screen Shot 2021-02-22 at 2 33 40 PM

File: _posts/2021-02-22-demo-day-8-contributing-psl.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+author: Jason DeBacker
+title: "Demo Day 7: How to contribute to PSL projects"
+---
+
+ {% include youtube.html content='https://youtu.be/LZUPYJ0RwJM' %}
+
+ ------
+
+ In the most recent PSL Demo Day, I illustrate how to contribute to the PSL projects.
+ The open source nature of projects in the PSL catalog allow anyone to contribute.
```
Comment (MaxGhenis) [inline _posts/2021-02-22-demo-day-8-contributing-psl.md:15]:
```suggestion
 The open source nature of projects in the PSL catalog allows anyone to contribute.
```

**Review by MaxGhenis — COMMENTED** [review]
Thanks @jdebacker, if #38 can be added tomorrow we can post this then too to preserve order.

File: _posts/2021-02-22-demo-day-8-contributing-psl.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+toc: true
+layout: post
+description: How to contribute to PSL projects
+categories: [demo-days, PSL, git, github]
+author: Jason DeBacker
+title: "Demo Day 7: How to contribute to PSL projects"
```
Comment (MattHJensen) [inline _posts/2021-02-22-demo-day-8-contributing-psl.md:7]: Demo Day 8

Comment (jdebacker) [reply _posts/2021-02-22-demo-day-8-contributing-psl.md]: Thanks for the catch @MattHJensen - fixed this in commit below.

**Comment by MaxGhenis** [comment]
Thanks @jdebacker, per today's call I updated the post date to today and removed the Demo Day number from the title. Merging.

**Pull request merged**